cli/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "deweb-cli",
+  "version": "0.2.0",
+  "description": "DeWeb is your gateway to the decentralized web, enabling seamless access to websites stored on-chain from any device, anywhere in the world.",
+  "license": "MIT",
+  "main": "./bin/index.js",
+  "bin": {
+    "deweb": "./bin/index.js"
+  },
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=7.0.0"
+  },
+  "scripts": {
+    "build": "tsc",
+    "start": "node ./bin/index.js",
+    "test": "jest --forceExit --detectOpenHandles",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "prettier": "prettier --check .",
+    "prettier:fix": "prettier --write .",
+    "fmt": "npm run lint:fix && npm run prettier:fix",
+    "fmt:check": "npm run lint && npm run prettier"
+  },
+  "dependencies": {
+    "@commander-js/extra-typings": "^12.1.0",
+    "@massalabs/massa-web3": "^5.0.0-next.20240829105104",
```

```suggestion
    "@massalabs/massa-web3": "^5.0.0-next.20240829105104",
```

maybe we just want ```^5.0.0-next```  and not the specific version